src/actions.ts

```diff
@@ -581,6 +585,129 @@ function meActions(
 				)
 			},
 		} satisfies CompanionActionDefinition,
+		[ActionId.TransitionSelectComponents]: {
+			name: 'Transition: Select components in transition',
+			options: [AtemMEPicker(model, 0), ...AtemTransitionSelectComponentsPickers(model)],
+			callback: async (action) => {
+				const mixeffect = getOptNumber(action, 'mixeffect')
+				const tp = getTransitionProperties(state.state, mixeffect)
+				if (!atem || !tp) return
+
+				const currentNextSelection = tp.nextSelection.slice().sort((a, b) => a - b)
```

Comment:
I don't think we need to be very concerned about performance here, instead of sorting this, you could use `currentNextSelection.includes(Enums.TransitionSelection.Background)`.
Keeps the code a bit simpler and easier to follow

Reply:
I agree there won't be so many (background + keys) that perf matters.  (At least assuming any Atem device has a human-workable number of mix effects, which I assume has to be the case!)

IMO, tho, this isn't about performance.  It's about computing next selection necessarily having to run through background and every single key.  The natural ordering for that, conceptually and in implementation, is background and then each key in sequence.  And if you have to run through them in that order -- isn't it simplest and clearest to just sort the selections to match processing in that order?

If this were changed to `includes`, the same change would have to happen to roughly lines 635, 682, and 695.  So you'd have to tolerate the complexity of four `includes` over only two sorts.  That seems worse than two conceptually simplifying sorts to me.

I'm going to update the PR not including this change for the moment, in the hope that I can persuade you this change would be undesirable.  If upon further consideration you still disagree, I'll make that additional change to it.  :-)

Reply:
Personally, I find it clearest to use `includes`, as then I don't need to consider the order. I don't think it makes it any less clear if considering the natural order either.
  
I questioned this because `currentNextSelection[i] === component` confused me at first. It isn't immediately obvious what `i` is, or what value I should expect it to have without looking at the other usages. 

I commented about performance, as it read like a reasonable optimisation to make at the cost of code readability so I assumed it would have been that